src/components/TextField/textfield.scss

```diff
@@ -11,6 +11,7 @@ $eccgui-size-textfield-padding-horizontal-small: $eccgui-size-inline-whitespace
 $eccgui-typo-textfield-fontweight: $eccgui-font-weight-regular !default;
 $eccgui-color-textfield-text: $eccgui-color-application-text !default;
 $eccgui-color-textfield-background: white !default; // TODO define global var for it
+$eccgui-color-codeeditor-background: $eccgui-color-textfield-background !default;
```

Comment:
`$eccgui-color-codeeditor-background` should be defined in the code editor stylesheet.